dashboard/app/controllers/aichat_controller.rb

```diff
@@ -47,6 +47,27 @@ def chat_completion
     sagemaker_response = AichatSagemakerHelper.request_sagemaker_chat_completion(input, params[:aichatModelCustomizations][:selectedModelId])
     latest_assistant_response = AichatSagemakerHelper.get_sagemaker_assistant_response(sagemaker_response)
 
+    filter_result = ShareFiltering.find_failure(latest_assistant_response, locale)
+    if filter_result&.type == ShareFiltering::FailureType::PROFANITY
+      # log here that profanity happened
```

Comment:
I want to log somewhere that this happened, but I'm not totally sure what is the right way to do that. [Discussing here](https://codedotorg.slack.com/archives/C046G4TRLEN/p1714672239480129), but didn't want to block review on this otherwise.

Reply:
Just confirming that logging wb part of this PR?

Reply:
Yep! Just added.